lib/api/kuzzle.js

```diff
@@ -234,6 +239,32 @@ class Kuzzle extends EventEmitter {
       });
   }
 
+   /** Manage errors to be thrown
+   *
+   * @param {...args} ...args
+   * @throws {KuzzleError}
+   */
+
+  throw(...args) {
+
+    const [domain, subdomain, errorName] = args;
+    const placeholders = args.length > 3 ? args.slice(3) : ' ';
+    const name = `${domain}-${subdomain}-${errorName}`;
+    if (!errorCodes.hasOwnProperty(domain)
```

Comment:
You can use lodash [get](https://lodash.com/docs/#get) method to retrieve the error and throw an error if its `undefined`

Reply:
I don't understand why and how it's better than using `hasOwnProperty`

Reply:
You can get subproperty by providing a path and without checking outer object existence
```
const error = _.get(`${domain}.${subdomain}.${errorName}`)
if (!error) {
  // throw
}
```

Reply:
ah nice, thankyu